Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/Actions/IElementTouchActions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Aquality.Appium.Mobile.Actions;
+using System.Drawing;
+
+namespace Aquality.Appium.Mobile.Elements.Actions
+{
+    public interface IElementTouchActions
```

Comment:
/// <summary>
        /// Describes Touch Actions for elements.
        /// </summary>